cobbler.spec

```diff
@@ -348,6 +363,9 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %{python3_sitelib}/cobbler/
 %{python3_sitelib}/cobbler-*
 %{_unitdir}/cobblerd.service
+%if 0%{?suse_version}
+%{_sbindir}/rccobblerd
+%endif
 %{tftpboot_dir}/*
 %{apache_dir}/cobbler
 %config(noreplace) %{_sharedstatedir}/cobbler
```

Comment:
This should probably not be `%config(noreplace)` I probably goofed here... :/

Reply:
Done